installers/common/MacInstaller.hs

```diff
@@ -286,6 +286,7 @@ buildElectronApp darwinConfig@DarwinConfig{dcAppName, dcAppNameApp} installerCon
       , "trezor-connect"
       , "trezor-link"
       , "typeforce"
+      , "util-deprecate"
```

Comment:
I see that this dependency already existed in yarn.lock before your PR. Is there a reason why do we need in the Installer?

Reply:
@danielmain yes because it is subdependency of trezor-connect (from the last update) and is necessary to declare it. Otherwise, you will get electron error.